helix-term/src/commands/lsp.rs

```diff
@@ -657,9 +657,20 @@ pub fn hover(cx: &mut Context) {
     );
 }
 pub fn rename_symbol(cx: &mut Context) {
-    ui::prompt(
+    let (view, doc) = current_ref!(cx.editor);
+    let primary_selection = doc
+        .selection(view.id)
+        .primary()
+        .fragment(doc.text().slice(..))
+        .to_string();
+    ui::prompt_with_input(
         cx,
         "rename-to:".into(),
+        if primary_selection.len() > 1 {
+            primary_selection
+        } else {
+            String::new()
+        },
```

Comment:
Why not just use `primary_selection`? Since it is already as `String`, even if it's empty we can use it, no?

Reply:
if user don't select the text in purpose, there is only one char in primary_selection. They might not realize to use the selection to prefill.
I am not sure about the case here, just think this will not change the default behavior. However I can make the change if you recommend that way.

Reply:
Oh, I guess that is unexpected as well, 1 character which currently they are on. I guess no.